### PR TITLE
Try: Material Web Components inside Calypso

### PR DESCRIPTION
**Pull request by blowery**
DO NOT MERGE

Well maybe.

### CODE REVIEW

**Comment by dmsnell** [comment]
sometime soon (or not) I will possibly not merge this.

**Comment by blowery** [comment]
@dmsnell thank you for that consideration

File: client/reader/following/main.jsx [hunk 1]
```diff
@@ -44,6 +45,9 @@ const FollowingStream = props => {
 		<Stream { ...props }>
 			{ config.isEnabled( 'reader/following-intro' ) && <FollowingIntro /> }
 			<MaterialButton className="following__button">Howdy</MaterialButton>
+			<MaterialTextField label="Dog">
```
Comment (dmsnell) [inline client/reader/following/main.jsx]: it's probably more accurate to put `Canine` - I'm concerned about losing semantic value with `Dog`

Comment (blowery) [reply client/reader/following/main.jsx]: "Howdy Dog" just reads better though.

Comment (dmsnell) [reply client/reader/following/main.jsx]: [You're the man now, dog!](
http://yourethemannowdog.ytmnd.com)

Comment (ockham) [reply client/reader/following/main.jsx]: Better check `canineuse.com` if browsers support this.

**Comment by alisterscott** [comment]
Do we want to keep this open?

**Comment by blowery** [comment]
Closing this down for now

**Pull request closed without merging**